app/Http/Controllers/ScoresController.php

```diff
@@ -54,6 +58,8 @@ public function download($rulesetOrSoloId, $id = null)
             'Content-Type' => 'application/x-osu-replay',
         ];
 
+        $score->user->statistics($ruleset, true)->increment('replay_popularity');
```

Comment:
there are some other places tracking replay counts that should be updated at the same time:
- monthly watch statistics - insert or increment `$score->user->replaysWatchedCounts` for the current `year_month`
- the replay's individual replay count - for legacy scores increment `$score->replayViewCount`, for lazer scores idk if it even exists...

Reply:
For what it's worth - as far as I can tell - the two above should be about it, looking at `osu-web-10` source. The point about `osu_replays_*` not really applying for lazer scores is valid though. Will probably need to be ran by @peppy (do we do a separate table? attempt to add a column to current? maybe put it in the `data` json blob?)

Reply:
I'll apply the first one, and the second one if it's a legacy score but will wait on a response for lazer scores.